orc/src/main/java/org/apache/iceberg/orc/ORC.java

```diff
@@ -124,18 +123,18 @@ private ReadBuilder(InputFile file) {
     /**
      * Restricts the read to the given range: [start, start + length).
      *
-     * @param start the start position for this read
-     * @param length the length of the range this read should scan
+     * @param newStart the start position for this read
+     * @param newLength the length of the range this read should scan
      * @return this builder for method chaining
      */
-    public ReadBuilder split(long start, long length) {
-      this.start = start;
-      this.length = length;
+    public ReadBuilder split(long newStart, long newLength) {
+      this.start = newStart;
+      this.length = newLength;
       return this;
     }
 
-    public ReadBuilder schema(org.apache.iceberg.Schema schema) {
-      this.schema = schema;
+    public ReadBuilder schema(org.apache.iceberg.Schema projectedSchema) {
```

Comment:
Nit: should probably be "projectSchema" not "projected". It doesn't make sense to use past tense; this is configuring a read that hasn't happened yet.

Reply:
fixed